src/components/SettingsEnableCAPTCHA.tsx

```diff
@@ -0,0 +1,71 @@
+import { Card, Loader, Notification, Toggle } from "@stellar/design-system";
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+
+import { ErrorWithExtras } from "@/components/ErrorWithExtras";
+
+import { useUpdateOrgCaptchaEnabled } from "@/apiQueries/useUpdateOrgCAPTCHAEnabled";
```

Comment:
Sorry, we also need to update the file names. Both in the `apiQueries` and `components` directories.